src/Behavioral.Automation/Elements/IWebElementWrapper.cs

```diff
@@ -49,6 +49,14 @@ public interface IWebElementWrapper
         /// <returns></returns>
         public IEnumerable<IWebElementWrapper> FindSubElements([NotNull] By locator, [CanBeNull] string caption);
 
+        /// <summary>
```

Comment:
Wait, maybe we should make a different interface. I'll tag you in BDD chat

Reply:
@vladimir-s-quantori Discussed in BDD chat. Is it good to go? :)